src/data.json

```diff
@@ -6,6 +6,14 @@
     "include": ["@slsa3"],
     "exclude": []
   },
+  "tssc-default": {
+    "name": "TSSC Default Stable",
+    "description": "Includes rules for levels 1, 2 & 3 of SLSA v0.1. This is the default config used by the RHTAP's TSSC templates.",
+    "environment": "rhtap",
+    "ecPoliciesRef": "tssc-stable",
```

Comment:
`tssc-stable` would be a branch or a tag that we'd create in the ec-policies repo.